packages/syft/src/syft/client/client.py

```diff
@@ -360,6 +361,9 @@ def api(self) -> SyftAPI:
 
         return self._api
 
+    def view_projects(self) -> List:
```

Comment:
We can have it has a property. Similar to users or code, etc.
 ```
    @property
    def users(self) -> Optional[APIModule]:
        if self.api is not None and hasattr(self.api.services, "user"):
            return self.api.services.user
        return None
```

Reply:
Great catch- added in https://github.com/OpenMined/PySyft/pull/7674/commits/e5868c32ab358ef37cf645f9a9213ceb46bedc46